espnet2/bin/enh_inference.py

```diff
@@ -253,6 +254,38 @@ def cal_permumation(self, ref_wavs, enh_wavs, criterion="si_snr"):
         _, perm = self.enh_model._permutation_loss(ref_wavs, enh_wavs, loss_func)
         return perm
 
+    @staticmethod
+    def from_pretrained(
+        enh_model_tag: Optional[str] = None,
+        **kwargs: Optional[Any],
+    ):
+        """Build SeparateSpeech instance from the pretrained model.
+
+        Args:
+            enh_model_tag (Optional[str]): Model tag of the pretrained models.
+                Currently, the tags of espnet_model_zoo are supported.
+
+        Returns:
+            SeparateSpeech: SeparateSpeech instance.
+
+        """
+        if enh_model_tag is not None:
+            try:
+                from espnet_model_zoo.downloader import ModelDownloader
+
+            except ImportError:
+                logging.error(
+                    "`espnet_model_zoo` is not installed. "
+                    "Please install via `pip install -U espnet_model_zoo`."
+                )
+                raise
+            d = ModelDownloader()
+            train_config, model_file = d.download_and_unpack(enh_model_tag).values()
+            # FIXME: The arugment name is mismatched, it is better to rename.
```

Comment:
@Emrys365 Here the arguments from downloader are different, which are defined in https://github.com/espnet/espnet/blob/fb75313bd1ddf027e075ea7197448c929b176f5d/espnet2/bin/pack.py#L24-L26
Maybe it is better to keep the same.

Reply:
Sorry for the mismatch. We should have kept consistency with other tasks. Could you remove the prefix "enh_" from all argument names in enh_inference.py?

Reply:
I will do it in the next PR.